src/reducers/contacts.reducer.js

```diff
@@ -0,0 +1,17 @@
+import { FETCH_CONTACTS } from '../actions/types';
+import { arrayToObject } from '../helpers/helper';
+import randomContactsList from '../data/data';
+
+const initialState = {
+  contacts: {},
+  favourites: []
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_CONTACTS:
+      return { ...state, contacts: arrayToObject(randomContactsList) };
```

Comment:
Here... this `arrayToObject` doesn't seem to have any sense. You convert array to objects here and then when you access the contacts you convert them back to array. Is there a reason for this?